src/big/math.cr

```diff
@@ -0,0 +1,15 @@
+require "big"
+
+module Math
```

Comment:
Please move this into `Math` overloads at the end of each of the respective `big_*.cr` files like [here](https://github.com/crystal-lang/crystal/blob/f0ec21e1209ba92ce67f4900a0a679cd3f92d0cd/src/big/big_float.cr#L280).